code/nuxeo-admin-center-jsf/src/main/java/org/nuxeo/connect/client/jsf/AppCenterViewsManager.java

```diff
@@ -651,7 +654,7 @@ public void setDevMode(boolean value) {
         params.put(Framework.NUXEO_DEV_SYSTEM_PROP, Boolean.toString(value));
         try {
             conf.saveFilteredConfiguration(params);
-            conf.getServerConfigurator().dumpProperties(conf.getUserConfig());
+            new ConfigurationMarshaller(System.getProperties()).dumpConfiguration(conf.getConfigurationHolder());
```

Comment:
Will this still dump only the "user" conf, and set aside the "default" conf? (as i see we still have distinct maps on the configuration holder for these)

Reply:
This behavior is pretty much the same, the configuration holder is more or less a wrapper of different `Properties` configuration.

Note that previously it didn't dump only the user conf, as the user conf inherited (by construction) from the default configuration.